rpc/core/tx.go

```diff
@@ -187,8 +187,8 @@ func TxSearch(query string, prove bool) ([]*ctypes.ResultTx, error) {
 		index := r.Index
 
 		if prove {
+			block := blockStore.LoadBlock(height)
 			// TODO: handle overflow
-			block := blockStore.LoadBlock(int(height))
 			proof = block.Data.Txs.Proof(int(index))
```

Comment:
can remove this `int`. but lets just wait till abci update

Reply:
should linter catch such things?

Reply:
not sure @zramsay . we dont have all linters active yet